_events/scheduled-talk.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: events
+title: Scheduled talk
+event-date: September 21th
+location: TBD
```

Comment:
This will be at 8th light.
> It probably best to leave the location line out if it's not known. The liquid template will simply leave of the "at" in the case of no location variable. 😄